### DISCUSSION TITLE
including R, engine:Julia

### DISCUSSION THREAD

**Discussion by Reulguen** [posit-dev/positron#4780] [Q&A]
Hi,
I'm trying to render a script that includes R chunks alongside Julia chunks. However, when I have both Julia and R chunks, it seems that I'm unable to set echo, warnings, etc., to FALSE in R, neither in the way it's done in Markdown nor with the #| syntax. Does anyone have a solution for this? Thanks!

**Comment by jmcphers** [comment]
This sounds like it's probably a problem with the Quarto extension rather than Positron itself. Do you see the same thing in VS Code with the Quarto extension installed?
